pom.xml

```diff
@@ -37,10 +44,8 @@
     <version.assertj>3.5.2</version.assertj>
     <version.jcip.annotations>1.0</version.jcip.annotations>
     <version.shrinkwrap.resolver>2.2.5</version.shrinkwrap.resolver>
+    <version.snakeyaml>1.17</version.snakeyaml>
```

Comment:
Ah I see - you have new tests for `Target` which are invoking it :) You should have pointed me to it right away and we wouldn't have this lengthy discussion. Sorry for the confusion.

Then let's just make sure we have versions aligned - to avoid potential runtime issues as we test against `1.17`, but Shrinkwrap is loading `1.15` :)